addons/web/static/src/views/relational_model.js

```diff
@@ -3482,6 +3485,31 @@ export class RelationalModel extends Model {
         if (this.defaultOrderBy && !(params.orderBy && params.orderBy.length)) {
             rootParams.orderBy = this.defaultOrderBy;
         }
+
+        if (rootParams.groupBy && rootParams.groupBy.length) {
+            const groupByFieldName = rootParams.groupBy[0].split(":")[0];
```

Comment:
why just groupBy[0]?

Reply:
you should surely look in search model fields... You have fetched them already, no?

Reply:
check property keys of the fake fields....